tailwind.config.cjs

```diff
@@ -2,7 +2,24 @@
 module.exports = {
   content: ["*.{html,ts}"],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        8: "2rem",
+        12: "3rem",
+      },
+      colors: {
+        transparent: 'transparent',
```

Comment:
most of these colors already exists within tailwind and you don't need to create custom colors for them

Reply:
Had problems picking up the tailwind style, decided to add custom fields. but fixed now thanks for the comment.